projects/admin/src/app/layout/dashboard/dashboard.component.ts

```diff
@@ -0,0 +1,134 @@
+import { Component} from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { FormBuilder, ReactiveFormsModule, Validators } from '@angular/forms';
+import { MatTableDataSource } from '@angular/material/table';
+import { MatCardModule } from '@angular/material/card';
+import { MatButtonModule } from '@angular/material/button';
+import { MatTableModule } from '@angular/material/table';
+import { CurrencyPipe } from '@angular/common';
+import { FormsModule } from '@angular/forms';
+import { MatInputModule } from '@angular/material/input';
+import { MatFormFieldModule } from '@angular/material/form-field';
+import { ProductModel } from '../../product.model';
+import { MasterService } from '../../service/master.service';
+import { MatSnackBar } from '@angular/material/snack-bar';
+
+@Component({
+  selector: 'app-dashboared',
+  standalone: true,
+  imports: [
+    FormsModule,
+    MatCardModule,
+    MatButtonModule,
+    MatFormFieldModule,
+    ReactiveFormsModule,
+    MatTableModule,
+    MatInputModule,
+    CurrencyPipe,
+    CommonModule,
+  ],
+  templateUrl: './dashboard.component.html',
+  styleUrl: './dashboard.component.css',
+})
+export class DashboardComponent  {
+  durationInSeconds = 5;
+
+
+  productlist?: ProductModel[];
+  datasource: any;
+  editdata?: ProductModel;
+  displayedColums: string[] = ['id', 'name', 'description', 'price', 'action'];
+
+  isadd = false;
```

Comment:
instead of 2 variables let's have one variable like form formStatus from enum with values, Normal, Add, Edit